portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Contact Rokiatou Sissoko | Information Science Student</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <script src="script.js"></script>
+  </head>
+  <body> 
+    <div class="top">
+    <header>
+    <img src="images/mintgreen.jpg">
+    </header>
+
+    <nav>
+        <ul>
+            <!-- <li><a href="index.html">Home</a></li> -->
```

Comment:
Generally it's good practice not to leave commented out code in a file. Can you remove this?